change/@fluentui-react-button-a033d25e-6861-40b4-a52e-eb4632568acb.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28589&commitId=08c7e015b3597a408bbce1aa2e29b36f2b96e583&env=prod&clientType=FLUENTUI)


  <details><summary><b>Button Converged</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button Converged.Size small - with long text wrapping.chromium.png|1| Changed|
|Button Converged.Size small.hover.chromium.png|2| Changed|
|Button Converged.Size small.default.chromium.png|1| Changed|
|Button Converged.Size small.pressed.chromium.png|2| Changed|

</details>


  <details><summary><b>CompoundButton Converged</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|CompoundButton Converged.Size small.hover.chromium.png|2| Changed|
|CompoundButton Converged.Size small - with long text wrapping.chromium.png|1| Changed|
|CompoundButton Converged.Size small.pressed.chromium.png|2| Changed|
|CompoundButton Converged.Size small.default.chromium.png|1| Changed|

</details>


  <details><summary><b>MenuButton Converged</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|MenuButton Converged.Size small - with long text wrapping.chromium.png|1| Changed|
|MenuButton Converged.Size small.default.chromium.png|1| Changed|
|MenuButton Converged.Size small.hover.chromium.png|2| Changed|
|MenuButton Converged.Size small.pressed.chromium.png|2| Changed|

</details>


  <details><summary><b>ToggleButton Converged</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ToggleButton Converged.Size small - with long text wrapping.chromium.png|1| Changed|
|ToggleButton Converged.Size small.hover.chromium.png|2| Changed|
|ToggleButton Converged.Size small.pressed.chromium.png|2| Changed|
|ToggleButton Converged.Size small.default.chromium.png|1| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>